code/datums/components/defibrillator.dm

```diff
@@ -100,6 +100,10 @@
 	on_cooldown = FALSE
 	SEND_SIGNAL(parent, COMSIG_DEFIB_READY)
 
+/datum/component/defib/proc/set_short_cooldown()
```

Comment:
Just make the set_cooldown() function take an optional number parameter instead of another proc, or give it a parameter like `short` or something